src/NServiceBus.Core/Serialization/SerializationConfigExtensions.cs

```diff
@@ -70,6 +72,20 @@ public static SerializationExtensions<T> AddDeserializer<T>(this EndpointConfigu
             return CreateSerializationExtension<T>(settings);
         }
 
+        /// <summary>
+        /// Disables dynamic type loading via `Type.GetType` to prevent loading of assemblies for types passed in message header `NServiceBus.EnclosedMessageTypes` to only allow message types during deserialization that were explicitly loaded.  
+        /// </summary>
+        public static void DisableDynamicTypeLoading<T>(this SerializationExtensions<T> config) where T : SerializationDefinition
```

Comment:
@timbussmann I've rebased this, don't think this requires any changes.

Reply:
The suggestion was to move this method (and `IsDynamicTypeLoadingEnabled` to https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/Serialization/SerializationExtensionsExtensions.cs because this extension class is targeting extension methods targeting `EndpointConfiguration`.